content/millennium/r4/clinical/care-provision/goal.md

```diff
@@ -21,10 +25,15 @@ The following fields are returned if valued:
 * [Subject (patient)](http://hl7.org/fhir/r4/goal-definitions.html#Goal.subject){:target="_blank"}
 * [Start Date](http://hl7.org/fhir/r4/goal-definitions.html#Goal.start_x_){:target="_blank"}
 * [Target Due Date](http://hl7.org/fhir/r4/goal-definitions.html#Goal.target.due_x_){:target="_blank"}
+  * [Reference](http://hl7.org/fhir/r4/references.html#Reference){:target="_blank"} ([Date](http://hl7.org/fhir/r4/datatypes.html#date){:target="_blank"})
 * [Status Date](http://hl7.org/fhir/r4/goal-definitions.html#Goal.statusDate){:target="_blank"}
 * [Expressed By](http://hl7.org/fhir/r4/goal-definitions.html#Goal.expressedBy){:target="_blank"}
+  * [Reference](http://hl7.org/fhir/r4/references.html#Reference){:target="_blank"} ([Patient](http://hl7.org/fhir/r4/patient.html){:target="_blank"} \| [Practitioner](http://hl7.org/fhir/r4/practitioner.html){:target="_blank"} \| [Related Person](http://hl7.org/fhir/r4/relatedperson.html){:target="_blank"})
+  * [Addresses](https://hl7.org/fhir/r4/goal-definitions.html#Goal.addresses){:target="_blank"}
+    * [Reference](http://hl7.org/fhir/r4/references.html#Reference){:target="_blank"} ([Condition](https://hl7.org/fhir/r4/condition-definitions.html){:target="_blank"})
 * [Note](http://hl7.org/fhir/r4/goal-definitions.html#Goal.note){:target="_blank"}
     * [Author](http://hl7.org/fhir/r4/goal-definitions.html#Goal.note){:target="_blank"}
+      * [Reference](http://hl7.org/fhir/r4/references.html#Reference){:target="_blank"} ([Text](http://hl7.org/fhir/r4/datatypes-definitions.html#Annotation.text){:target="_blank"} \| [Time](http://hl7.org/fhir/r4/datatypes-definitions.html#Annotation.time){:target="_blank"})
```

Comment:
+1. Also should we try and add the actual names of the fields, like authorReference?

Reply:
> Also should we try and add the actual names of the fields, like authorReference?

So... That _would_ be my preference, but it's also inconsistent with every one of these lists in all of the resources. For now I would say no so that these fit in with what's already there, and to not blow out scope. But I do think we need to revisit how this info is conveyed. I not a fan of how we're hiding the field names and just the wild inconsistency in general.